apps/extension/src/ui/apps/popup/pages/Portfolio/PortfolioAssets.tsx

```diff
@@ -114,8 +123,17 @@ const PageContent = ({
               </div>
               <AccountTypeIcon className="text-primary" origin={account?.origin} />
             </div>
-            <div className="text-md overflow-hidden text-ellipsis whitespace-nowrap">
-              <Fiat amount={balances.sum.fiat("usd").total} isBalance />
+            <div
+              className={classNames(
+                "overflow-hidden text-ellipsis whitespace-nowrap",
+                account ? "text-body-secondary" : "text-md"
+              )}
+            >
+              {account ? (
+                <Address address={formattedAddress} />
```

Comment:
thinking it would be more practical if the <Address /> component was resolving addresses on it's own, with a flag to enable or disable this.
We got addresses all over the place, and will have many more in the future, so it would be nice not to have to resolve chainIds upfront each time we want to display an address, nor even to have to think about it.

Reply:
I think I don't understand what you mean. It looks to me like the `<Address />` component *is* resolving addresses on its own?

Reply:
right nevermind, thought the useFormattedAddress call that is often called upfront was doing some resolving